client/state/sharing/publicize/actions.js

```diff
@@ -42,21 +42,33 @@ export function sharePost( siteId, postId, skippedConnections, message ) {
 			message,
 		} );
 
-		return new Promise( ( resolve ) => {
-			wpcom
-				.undocumented()
-				.publicizePost( siteId, postId, message, skippedConnections, ( error, data ) => {
-					// Note: successes are recorded in data.results, errors are recorded in data.errors. There could be
-					// several errors and several successes.
-					if ( error || ! data.results ) {
-						dispatch( { type: PUBLICIZE_SHARE_FAILURE, siteId, postId, error } );
-					} else {
-						dispatch( { type: PUBLICIZE_SHARE_SUCCESS, siteId, postId } );
-					}
+		const body = {
+			skipped_connections: [],
```

Comment:
The property value could be `skippedConnections ?? []`?

Reply:
Good point. I've double-checked where we trigger `sharePost`, and we'll always use an array, so this safeguard is more than enough. Addressed in 57b224b33c4a5d4ec39e3af6ac43c5aae4c121da.